### PR TITLE
enable self-diagnostics in example app

### PR DESCRIPTION
**Pull request by TimothyMothra**
The self-diagnostics feature has been invaluable while testing.
I would like to enable this in our example app.

I confirmed that the *.log files are excluded from git.

### CODE REVIEW

File: examples/WebApp.AspNetCore/WebApp.AspNetCore.csproj [hunk 1]
```diff
@@ -8,4 +8,11 @@
     <ProjectReference Include="..\..\NETCORE\src\Microsoft.ApplicationInsights.AspNetCore\Microsoft.ApplicationInsights.AspNetCore.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <!--This file enables self-diagnostics-->
```
Comment (cijothomas) [inline examples/WebApp.AspNetCore/WebApp.AspNetCore.csproj:12]: perhaps add a link to self-diagnostics doc here.

File: examples/WebApp.AspNetCore/ApplicationInsightsDiagnostics.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+﻿{
+  "LogDirectory": ".",
+  "FileSize": 1024,
+  "LogLevel": "Verbose"
```
Comment (cijothomas) [inline examples/WebApp.AspNetCore/ApplicationInsightsDiagnostics.json:4]: Warning would be sufficient for most cases, unless investigating something very specific.

**Review by rajkumar-rangaraj — APPROVED** [review]

**Review by cijothomas — APPROVED** [review]
LGTM.
Do we have asp.net sample apps and worker/console as well?

**Comment by TimothyMothra** [comment]
> LGTM.
> Do we have asp.net sample apps and worker/console as well?

@cijothomas not yet, but this is something I want to add! :)

**Pull request merged**